### DISCUSSION TITLE
move item from the more options menu to the main menu

### DISCUSSION THREAD

**Discussion by Sebo1090** [moudey/Shell#289] [Q&A]
Is it possible to move item from the more options menu to the main menu

**Comment by moudey** [comment]
In the shell file, disable static item modifications like the following example
```
static
{	
    // import 'imports/static.nss'
    // item(where=this.title.length > 25 menu=title.more_options)
}
```
Save changes, press <kbd>CTRL</kbd> + <kbd>RIGHT-CLICK</kbd> to reload settings.

**Comment by Sebo1090** [comment]
thank you finally programs are after cataloging
